comp/rdnsquerier/impl/config.go

```diff
@@ -0,0 +1,103 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+package rdnsquerierimpl
+
+import (
+	"github.com/DataDog/datadog-agent/comp/core/config"
+	"github.com/DataDog/datadog-agent/comp/core/log"
+)
+
+type rdnsQuerierConfig struct {
+	enabled  bool
+	workers  int
+	chanSize int
+
+	rateLimiterEnabled bool
+	rateLimiterLimit   int
+	rateLimiterBurst   int
+
+	cacheEnabled         bool
+	cacheEntryTTL        int
+	cacheCleanInterval   int
+	cachePersistInterval int
+
+	// debug - TODO remove
+	fakeResolver                 bool
+	generateFakeQueriesPerSecond int
+	lookupDelayMs                int
+}
+
+func newConfig(agentConfig config.Component, logger log.Component) *rdnsQuerierConfig {
+	netflowRDNSEnrichmentEnabled := agentConfig.GetBool("network_devices.netflow.reverse_dns_enrichment_enabled")
+
+	c := &rdnsQuerierConfig{
+		enabled:  netflowRDNSEnrichmentEnabled,
+		workers:  agentConfig.GetInt("reverse_dns_enrichment.workers"),
+		chanSize: agentConfig.GetInt("reverse_dns_enrichment.chan_size"),
+
+		rateLimiterEnabled: agentConfig.GetBool("reverse_dns_enrichment.rate_limiter.enabled"),
+		rateLimiterLimit:   agentConfig.GetInt("reverse_dns_enrichment.rate_limiter.limit"),
```

Comment:
This is per-second right?

Reply:
Yes, I was following the naming of the underlying rate limiter, but it is vague.  Maybe it should be changed to `reverse_dns_enrichment.rate_limiter.limit_per_sec`.  Thoughts @FlorianVeaux ?

Reply:
Changed to `limit_per_sec` in https://github.com/DataDog/datadog-agent/pull/27170/commits/4fc8f9d04ad8f16860775f78cda7f6a6d675613a

Reply:
👍